graphene/sys/tests/abi.rs

```diff
@@ -2,6 +2,8 @@
 // from gir-files (https://github.com/gtk-rs/gir-files)
 // DO NOT EDIT
 
+#![allow(unused)]
```

Comment:
Discussion about this one is https://github.com/gtk-rs/gtk-rs/pull/30#discussion_r525060701

@bilelmoussaoui You want to look into this, or we keep it like above for now and create yet another gir issue for later? :)

Reply:
Let's keep it as is and create an issue for gir